### ISSUE TITLE
Duplicating Parameters links in build

### ISSUE THREAD

**Issue by dpogue** [jenkinsci/ghprb-plugin#350]
After updating the plugin (on Jenkins 1.656), all my pull requests jobs are failing and I'm seeing some strange duplication of the parameters link in the sidebar.

<img width="208" alt="screen shot 2016-05-24 at 15 56 45" src="https://cloud.githubusercontent.com/assets/241708/15522600/81001a9c-21c8-11e6-9869-dc09c40c040a.png">

This is the same issue reported by @orrc in #344.

**Comment by juhakuu** [comment]
I'm also seeing this issue after updating the plugin from 1.31.4 to 1.32.1 on Jenkins 1.642.1.

**Comment by DavidTanner** [comment]
Please check with latest, 1.32.2

**Comment by juhakuu** [comment]
Upgrading to 1.32.2 fixed this issue for me. #352 still persists, though.

**Comment by alexwhitman** [comment]
Upgrading to 1.32.2 fixed it for me with new builds.  Pre-upgrade builds still show multiple links but I can understand that.

However, clicking on the Parameters link doesn't actually show any parameters that were supplied to the build.

**Comment by juhakuu** [comment]
I just noticed the same - parameters listing is now empty, while pre-1.32.1 it actually listed the gotten parameters.

**Comment by benpatterson** [comment]
This was fixed in build 1.33.4. Closing.

**Comment by rajasekaran07** [comment]
@benpatterson still I am facing this issue. I am using jenkins 2.32.1 and I passed my upstream jobs parameters to downstream job. But I am unable to access those parameter in my downstream job.

Steps I followed.

![image](https://cloud.githubusercontent.com/assets/19193992/21839184/67526c72-d7fc-11e6-8fcd-af37d4e839cb.png)

In my downstream job nothing is displaying under parameter:

![image](https://cloud.githubusercontent.com/assets/19193992/21839204/80a7382e-d7fc-11e6-8fcd-389c9f14be87.png)

Kindly help me out.

--Raja
